### ISSUE TITLE
QR Code não conecta Whatapp

### ISSUE THREAD

**Issue by dantaspaulo** [riservato-xyz/whaticket-installer#8]
Essa sua instalação é muito boa e pra falar a verdade, foi a única que consegui colocar pra funcionar em pouco tempo.

Problema: Após a instalação, ao ler o QR Code para iniciar uma conexão, o Whatsapp conecta no Celular, mostrando uma conexão ativa, porém, no Whaticket, nada acontece!
Ou seja, não tem dado para conectar efetivamente e utilizar, embora a instalação esteja linda.
Vi umas discursão no fórum, mas tudo bem confuso.

**Comment by silviorseara** [comment]
também estou com o mesmo problema....você encontrou alguma solução?

**Comment by dantaspaulo** [comment]
> também estou com o mesmo problema....você encontrou alguma solução?

Cara, eu instalei outro repositório e funcionou. Mas eu percebi que quando vc conecta o QR, precisa esperar uns 30 a 40 segundo sem tocar em nada na tela, pra ele poder sincronizar. Ou seja, não é instântaneo, vai depender da capacidade da sua VPS, da sua maquina, etc... por isso tem esse delay

**Comment by silviorseara** [comment]
verdade...aqui deu certo tbm, mas de fato não é instantâneo e precisa fechar a tela do QR Code para ele conectar.

**Comment by dev-stupid-codes** [comment]
Essa questão do QR code demorar depende de quantas mensagens ativas você possui no seu whatsapp, também. Resolveram?

@dantaspaulo
